netket/driver/abstract_variational_driver.py

```diff
@@ -190,7 +191,7 @@ def run(
         save_params_every=50,  # for default logger
         write_every=50,  # for default logger
         step_size=1,  # for default logger
-        callback=lambda *x: True,
+        callback=InvalidLossStopping,
```

Comment:
Providing this as the default callback has the downside that if users pass their own callbacks, they can inadvertently remove the invalid loss stopping.

One option to still include it by default (and allow users to remove it) would be to set the default `callback=[]` and add something like
```python3
if stop_on_invalid_loss and all(not isinstance(cb, InvalidStopLoss) for cb in callback):
    callback.append(InvalidStopLoss())
```
depending on a new argument `stop_on_invalid_loss: bool`.